src/ts/ConvivaAnalytics.ts

```diff
@@ -292,6 +299,12 @@ export class ConvivaAnalytics {
   };
 
   private onPlaying = (event: any) => {
+    // When seeking over the buffer the player emits an onPlaying event followed by an onStallStarted within 0 ms.
+    // To suppress this play state transition we check the buffer if the video element is really ready to start playing.
+    if (this.player.getVideoBufferLength() === 0) {
```

Comment:
what if it is an audio only stream?

Reply:
Good point. 

So is for an audio only stream the `getVideoBufferLength` always 0 (and vice versa)?

Reply:
Since this is a bug in the player that we need to fix, this check should probably be removed?

Reply:
Yes, I'm waiting for a response of conviva but it will be removed anyway.